### PR TITLE
Fixed handling of state updates in system profiles

### PR DESCRIPTION
**Pull request by cweitkamp**
- Fixed handling of state updates in system profiles

See https://github.com/eclipse/smarthome/pull/6451#issuecomment-439598333

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

**Comment by pfink** [comment]
The FollowProfile should stay as it is, here this is the intended behavior. Fix for OffsetProfile LGTM.

File: bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/profiles/SystemFollowProfile.java [hunk 1]
```diff
@@ -53,12 +50,7 @@ public ProfileTypeUID getProfileTypeUID() {
 
     @Override
     public void onStateUpdateFromItem(State state) {
-        if (!(state instanceof Command)) {
-            logger.debug("The given state {} could not be transformed to a command", state);
-            return;
-        }
-        Command command = (Command) state;
-        callback.handleCommand(command);
+        callback.handleUpdate(state);
```
Comment (htreu) [inline bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/profiles/SystemFollowProfile.java]: I agree with @pfink, the follow profile will forward state updates from an item to the following item as commands. Please revert.

**Comment by cweitkamp** [comment]
Done. Thanks. Maybe I should read the JavaDoc next time.

**Comment by maggu2810** [comment]
@htreu As you have requested changes, you have to revoke your veto (or request further changes).

**Comment by htreu** [comment]
Restarted Travis.

**Pull request merged**